vizro-core/docs/pages/user-guides/graph.md

```diff
@@ -72,7 +71,7 @@ To add a [`Graph`][vizro.models.Graph] to your page, do the following:
 
     [Graph]: ../../assets/user_guides/components/graph.png
 
-Note that in the above example we directly inserted the chart into the `figure` argument for the `.py` version. This is also the simplest way to connect your chart to a Pandas `DataFrame` - for other connections, please refer to [this guide on data connections](data.md). For the `yaml` version, we simply referred to the [`plotly.express`](https://plotly.com/python/plotly-express/) name by string.
+In the Python example we directly inserted the pandas DataFrame `df` into `figure=px.scatter_matrix(df, ...)`. This is [the simplest way to connect a chart to data](data.md#supply-directly). For the YAML version, we [refer to the data source by name as `data_frame: iris`](static-data.md#reference-by-name). For a full explanation of the different methods you can use to provide data to your dashboard, see [our guide to using data in Vizro](data.md).
```

Comment:
It seems like relative link `static-data.md#reference-by-name` has to be changed.